.github/workflows/publish.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - run: echo "$DATA"
         env:
           DATA: ${{ toJson(github.event.head_commit) }}
-      - uses: actions/checkout@v2-beta
+      - uses: actions/checkout@v3
```

Comment:
It looks like in the other scripts you used v2, should we use the same script in all the scripts or this is done this way for a reason?

Reply:
Nice catch! I need to upgrade all of those into v3